src/nvim/os/env.c

```diff
@@ -557,6 +557,27 @@ char *vim_getenv(const char *name)
     if (p_hf != NULL && vim_strchr(p_hf, '$') == NULL) {
       vim_path = (char *)p_hf;
     }
+
+#ifdef WIN32
+    // Find runtime path relative to the nvim binary i.e. ../share/runtime
+    char exe_name[MAXPATHL];
+    char abs_runtime_path[MAXPATHL];
+    if (vim_path == NULL) {
+      if (os_exepath(exe_name, MAXPATHL) == 0) {
+        // Remove trailing bin/nvim.exe
```

Comment:
But it is actually removing the last two path components (presumably bin/nvim.exe) or rather its removing the basename twice.